### DISCUSSION TITLE
Example sentence for はじめまして is completely cooked

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#16] [Errors and Corrections]
Japanese:

> 「はじめまして、ジェイムズさん」「こちらこそ、あなたにお会いできて、こんなに嬉しいことはありません」

English:

> "How do you do, Mr James?" "Fine, thanks for asking. Nice to meet you."

Honestly I can't even begin to imagine how I'd fix this sentence. It's just... lol.

**Comment by Morgawr** [comment]
Oh looks like it's also the same example sentence for こちらこそ

**Comment by stephenmk** [comment]
Surprisingly, there are only two other sentences in the corpus for はじめまして. Not a lot of great options to choose from.

The Japanese sentence seems mostly okay to me. Overuse of "あなた" is a red flag, but there's always a lot of that in example sentences.

I updated the translation:

> "Pleased to meet you, Mr. James." "The pleasure is all mine. It's an honor to make your acquaintance."
